package.json

```diff
@@ -12,7 +12,7 @@
     "test:api": "jest --colors --verbose",
     "test:clean-reports": "rm -rf ./results",
     "test:debug": "cross-env NODE_ENV=debug ./start-tests.sh",
-    "test:headed": "cross-env NODE_ENV=development LIVE_MODE=true BROWSER=chrome ./start-tests.sh",
+    "test:headed": "cross-env NODE_ENV=development LIVE_MODE=true ./start-tests.sh",
```

Comment:
By default, this is already set to chrome. If the end-user wants to test with another browser, they will need to export and set the `BROWSER` variable.